.github/workflows/label_and_milestone_checker.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           PR_AFFILIATION: ${{ github.event.pull_request.user.company }}
         run: |
           echo "PR author: $PR_AUTHOR"
-          if [ "$PR_AUTHOR" = "napari-bot" ] || [ "$PR_AUTHOR" = "pre-commit-ci" ] || [ "$PR_AUTHOR" = "dependabot" ]; then
+          if [[ "$PR_AUTHOR" == *"[bot]" ]]; then
```

Comment:
```suggestion
          if [[ "$PR_AUTHOR" == *"[bot]" || "$PR_AUTHOR" == *-bot ]]; then
```
I think we still want to catch `napari-bot` right?

Reply:
napari-bot will appear as `napari-bot[bot]` in here, so it will be caught!

Reply:
It didn't appear that way in the pr I checked...